app/serializers/doi_serializer.rb

```diff
@@ -2,7 +2,7 @@ class DoiSerializer < ActiveModel::Serializer
   include Bolognese::Utils
   include Bolognese::DoiUtils
 
-  attributes :doi, :prefix, :suffix, :identifier, :url, :creator, :title, :publisher, :resource_type_subtype, :description, :version, :metadata_version, :schema_version, :state, :is_active, :reason, :landing_page, :xml, :published, :registered, :updated
+  attributes :doi, :prefix, :suffix, :identifier, :url, :creator, :title, :publisher, :resource_type_subtype, :description, :version, :metadata_version, :schema_version, :state, :is_active, :reason, :landing_page, :published, :registered, :updated
```

Comment:
But don't we need this in bolognesa and content-negotiation ?https://github.com/datacite/bolognese/blob/62e94949d4363cb55bfe9cf4e97715b940866422/lib/bolognese/readers/datacite_reader.rb#L18

Reply:
I added it back in for compatibility. But I want to rewrite how bolognese and content-negotiation are fetching metadata - content negotiation is too slow, and the main reason is always this metadata conversion overhead.

Reply:
I actually think the bug I was fixing is partially related to this part. Because LUPO seem to regenerate XML using Bolognese and put it in this field in the serializer and Bolognese calls LUPO to get the XML from the same field. In some cases its like a circular relationship.

Reply:
Content negotiation would be faster if we either fetch crosscite.json or datacite.xml from `lupo`, depending on the request.

Reply:
`bracco` isn't using the `xml` attribute in the `lupo` API response anymore, but instead uses content negotiation for DataCite XML.

Reply:
@kjgarza yes, it is convoluted and circular, as `bolognese` fetches DataCite XML from `lupo` which uses `bolognese`...